admin/src/Messages/MessageShow.js

```
@@ -1,74 +1,66 @@
-                    <h3
-                        className="uk-panel-title"
-                        dangerouslySetInnerHTML={{ __html: message.subject }}
-                    />
-                </div>
-                <div className="uk-panel uk-panel-box uk-margin-bottom">
-                    <div dangerouslySetInnerHTML={{ __html: message.body }} />
-                </div>
+            <div className="uk-panel uk-panel-box uk-margin-bottom">
+                <h3
+                    className="uk-panel-title"
+                    dangerouslySetInnerHTML={{ __html: message.subject }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)